sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -65,12 +63,13 @@
 iris.target = iris.target[perm]
 
 
-def check_classification_toy(loss):
+def check_classification_toy(presort, loss):
```

Comment:
@sameshl  ,remove the presort parameter from the function arguements. function should only take one arguement **loss**, It has also the same reason that presort is going to be depricated very soon.